blathering-barnacles/Bangazon
# Ticket11

Description: Added the ability to delete a Training Program from the detailed view of that specific Training Program. Also added a back button on the Program Form that takes you back to the detail view of that Training Program, and a back button there also that takes you back to the list of Training Programs.
Also added the corresponding tests that test this deleting feature. Check them out in the tests folder in the test_trainingProgram.py.

Steps to Manual Testing:
* Click on a Training Program from the Training Programs list. 
* Click on the delete this Training Program Button located at the bottom of the detailed view url.
* The Training Program should now be deleted from your database, check your database to make sure. 
*Run python manage.py test and make sure all tests are working
Link to Feature Ticket:  [Ticket11](https://github.com/blathering-barnacles/Bangazon/issues/11)
